boards/raspberrypi/rpi_pico2/doc/index.rst

```diff
@@ -0,0 +1,92 @@
+.. _rpi_pico2:
+
+Raspberry Pi Pico 2
+###################
+
+Overview
+********
+
+The Raspberry Pi Pico 2 is the second-generation product in the Raspberry Pi
+Pico family. From the `Raspberry Pi website <https://www.raspberrypi.com/documentation/microcontrollers/pico-series.html>`_ is referred to as Pico 2.
+
+.. figure:: img/pico-2.jpg
+   :align: center
+   :alt: Raspberry Pi Pico 2
+
+   Raspberry Pi Pico 2 (Credit: Raspberry Pi Ltd)
+
+Hardware
+********
+
+- Dual Cortex-M33 or Hazard3 processors at up to 150MHz
+- 520KB of SRAM, and 4MB of on-board flash memory
+- USB 1.1 with device and host support
+- Low-power sleep and dormant modes
+- Drag-and-drop programming using mass storage over USB
+- 26 multi-function GPIO pins including 3 that can be used for ADC
+- 2 SPI, 2 I2C, 2 UART, 3 12-bit 500ksps Analogue to Digital - Converter (ADC), 24 controllable PWM channels
+- 2 Timer with 4 alarms, 1 AON Timer
+- Temperature sensor
+- 3 Programmable IO (PIO) blocks, 12 state machines total for custom peripheral support
+   - Flexible, user-programmable high-speed IO
+   - Can emulate interfaces such as SD Card and VGA
+
+Supported Features
+==================
+
+The rpi_pico board configuration supports the following
```

Comment:
rpi_pico isn't even right, needs full name as used for west build with \`\` around it and this is a board target, not a configuration

Reply:
> rpi_pico isn't even right, 

Well spotted, corrected.

> needs full name as used for west build with `` around it.

Please can you raise an issue against the documentation? This is not covered in the Board Porting Guide, or the template it links to at doc/templates/board.tmpl

> and this is a board target, not a configuration

This phrase is used literally hundreds of times:
```shell
$ grep -r -m1 "board configuration supports the following" ./boards/ | wc -l
348
```
Including in Nordic's files:
```shell
$ grep -r -m1 "board configuration supports the following" ./boards/nordic
./boards/nordic/nrf21540dk/doc/index.rst:The nrf21540dk/nrf52840 board configuration supports the following
./boards/nordic/nrf51dk/doc/index.rst:The nrf51dk/nrf51822 board configuration supports the following nRF51
./boards/nordic/nrf51dongle/doc/index.rst:The nrf51dongle/nrf51822 board configuration supports the following nRF51
./boards/nordic/nrf52833dk/doc/index.rst:The ``nrf52833dk/nrf52833`` board configuration supports the following
./boards/nordic/nrf52840dk/doc/index.rst:The ``nrf52840dk/nrf52840`` board configuration supports the following
./boards/nordic/nrf52840dongle/doc/index.rst:The ``nrf52840dongle/nrf52840`` board configuration supports the following
./boards/nordic/nrf52dk/doc/index.rst:The ``nrf52dk/nrf52832`` board configuration supports the following
./boards/nordic/nrf5340dk/doc/index.rst:The ``nrf5340dk/nrf5340/cpuapp`` board configuration supports the following
./boards/nordic/nrf54h20dk/doc/index.rst:The ``nrf54h20dk/nrf54h20/cpuapp`` board configuration supports the following
./boards/nordic/nrf54l15pdk/doc/index.rst:The ``nrf54l15pdk/nrf54l15/cpuapp`` board configuration supports the following
./boards/nordic/nrf7002dk/doc/index.rst:The ``nrf7002dk/nrf5340/cpuapp`` board configuration supports the following hardware features:
./boards/nordic/nrf9131ek/doc/index.rst:The ``nrf9131ek/nrf9131`` board configuration supports the following
./boards/nordic/nrf9151dk/doc/index.rst:The ``nrf9151dk/nrf9151`` board configuration supports the following
./boards/nordic/nrf9160dk/doc/index.rst:The nrf9160dk/nrf9160 board configuration supports the following
./boards/nordic/nrf9161dk/doc/index.rst:The ``nrf9161dk/nrf9161`` board configuration supports the following
```
Some of which also don't have the double backticks around them:
```shell
$ grep -r -m1 "The n.*board configuration supports the following" ./boards/nordic
./boards/nordic/nrf21540dk/doc/index.rst:The nrf21540dk/nrf52840 board configuration supports the following
./boards/nordic/nrf51dk/doc/index.rst:The nrf51dk/nrf51822 board configuration supports the following nRF51
./boards/nordic/nrf51dongle/doc/index.rst:The nrf51dongle/nrf51822 board configuration supports the following nRF51
./boards/nordic/nrf9160dk/doc/index.rst:The nrf9160dk/nrf9160 board configuration supports the following
```

Reply:
It literally is: https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#board-terminology

It is present in older boards because the term was introduced with hwmv2, before hwmv2 it did not exist, and given that hwmv2 involved a porting of every single soc and board in zephyr by very few people in a very limited amount of time, there wasn't time to go through and update documentation.

Reply:
This has been changed, and a spearate issue (and PR) have been raised to cover improving existing files.